json-to-go.js

```diff
@@ -16,7 +16,7 @@ function jsonToGo(json, typename)
 
 	try
 	{
-		data = JSON.parse(json);
+		data = JSON.parse(json.replace(/\.0/g, ".1"));
```

Comment:
Ah, you're right, somehow glossed over that. I guess this is acceptable. If people complain, we can figure something else out (maybe).